src/main.rs

```diff
@@ -270,6 +270,17 @@ macro_rules! eat {
     };
 }
 
+/// This macro takes any expression which evaluates to a Result, returns the Ok value, or continues in case of an Err.
```

Comment:
Let's add a bit of syntax highlighting :)

```suggestion
/// This macro takes any expression which evaluates to a `Result`, returns the `Ok` value, or continues in case of an `Err`.
```